<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessDialog</class>
 <widget class="QDialog" name="ProcessDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game list</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonRefresh">
       <property name="text">
        <string>&amp;Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editFilter">
       <property name="placeholderText">
        <string>Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonFilter">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>&amp;Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAddRule">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Add rule</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Processes:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="editProcesses">
     <property name="font">
      <font>
       <family>Courier</family>
      </font>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Game rules:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="editRules">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Courier</family>
      </font>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
